### PR TITLE
Fix #10683 - Introduce Issue Forms with YAML

### LINKED ISSUE #10683: Make use of GitHub issue forms
There is now the possibility to create "Issue forms" (currently in beta). They (optionally) can force the user to enter more information or at least makes it way easier for users. Research has shown, that these forms improve the issue content quality a lot (win-win situation)

Microsoft Powertoys is using it, and it feels pretty cool creating a new issue with these forms 🤩

Example: https://github.com/microsoft/PowerToys/blob/master/.github/ISSUE_TEMPLATE/bug_report.yml

grafik

### PR DESCRIPTION
**Pull request by Lineflyer**
Mainly migrating the exiting template into a form.
Further improvements welcome after merging.

### CODE REVIEW

**Comment by Lineflyer** [comment]
image

Example Bug Report (screenshot cut off):
image

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      label: Expected result after these steps?
+      description: Describe what you expected to happen instead (correct behavior).
+      placeholder: "
+      Examples:\n
+      The app does not crash\n
+      Its showing the value ab\n
+      The dialog is shown
+      "
+    validations:
+      required: true
```
Comment (fm-sys) [inline .github/ISSUE_TEMPLATE/bug_report.yml:61]: Here as well...

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    attributes:
+      label: What solution would you suggest?
+      description: Describe your sugggested solution and the way it should be implemented.
+      placeholder: "
+      Examples:\n
+      There should be a new menu item xyz in menu abc, which performs [...]\n
+      Change the existing function to include [...]
+      "
+    validations:
+      required: true
```
Comment (fm-sys) [inline .github/ISSUE_TEMPLATE/feature_request.yml:35]: Really required? Maybe same as for bugs...

Some things can  be explained in one sentence, other things need further explanation... 

I don't think it makes sense to technically force users to enter something. They can still enter a "-" which doesn't help us either. I'm quite sure that users will use the text box if it makes sense, independent of whether it is required or not.

**Review by fm-sys — COMMENTED** [review]
Looks really good already, thanks for stepping into it 👍

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+      label: Actual result after these steps?
+      description: Describe what actually happens after the steps above (wrong behavior).
+      placeholder: "
+      Examples:\n
+      The app crashes\n
+      Its showing the value xy\n
+      Nothing happens
+      "
+    validations:
+      required: true
```
[the remark this answers is not in the document]
Comment (Lineflyer) [reply .github/ISSUE_TEMPLATE/bug_report.yml:48]: Might be true. I will set it to false for now.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (Lineflyer) [reply .github/ISSUE_TEMPLATE/bug_report.yml:61]: OK, see above

File: .github/ISSUE_TEMPLATE/config.yml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: true
 contact_links:
   - name: 3. Contact our c:geo support team
-    url: https://www.cgeo.org/support
-    about: If you have questions regarding the usage of c:geo you should contact our support team by mail instead. Click to open our support page for more information.
+    url: https://mailxto.com/ft2ql4
```
[the remark this answers is not in the document]
Comment (Lineflyer) [reply .github/ISSUE_TEMPLATE/config.yml:4]: Unfortunately mailto links are not supported the last time I tried it. Therefore I used that mailto shortener...not sure how reliable it is....but I guess this way is rarely used anyway.

[the remark this answers is not in the document]
Comment (fm-sys) [reply .github/ISSUE_TEMPLATE/config.yml:4]: Ok, fine for me...

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 2 above)
Comment (Lineflyer) [reply .github/ISSUE_TEMPLATE/feature_request.yml:35]: OK, I will set it to false.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (Lineflyer) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Done

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 2 above)
Comment (Lineflyer) [reply .github/ISSUE_TEMPLATE/feature_request.yml]: Done

**Comment by fm-sys** [comment]
From my side ready to merge 👍 I'm really looking forward to try it out 😁

If there should be any problems, we can still make adjustments. "Es ist ja nichts in Stein gemeißelt" ;)

**Pull request merged**